recipes/arpack/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+else
+  SO_EXT='.so'
+fi
+
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_SHARED_LIBS=ON \
```

Comment:
That'd be doable, but frankly I don't see the point of providing statics.
Do we want all library packages to provides both then ?